lib/vmfloaty/utils.rb

```diff
@@ -79,40 +79,62 @@ def self.generate_os_hash(os_args)
   end
 
   def self.pretty_print_hosts(verbose, service, hostnames = [], print_to_stderr = false)
+    fetched_data = self.get_host_data(verbose, service, hostnames)
+    fetched_data.each do |hostname, host_data|
+      case service.type
+      when 'ABS'
+      # For ABS, 'hostname' variable is the jobID
+        host_data['allocated_resources'].each do |vm_name, _i|
+          puts "- [JobID:#{host_data['request']['job']['id']}] #{vm_name['hostname']} (#{vm_name['type']}) <#{host_data['state']}>"
```

Comment:
These puts are probably ok to keep just because it's doing some specific formatting as a utils class. :+1: